wgengine/netstack/netstack_test.go

```diff
@@ -39,12 +39,12 @@ func TestInjectInboundLeak(t *testing.T) {
 	if !ok {
 		t.Fatal("not an InternalsGetter")
 	}
-	tunWrap, magicSock, ok := ig.GetInternals()
+	tunWrap, magicSock, d, ok := ig.GetInternals()
```

Comment:
same, d is a big nondescript unless I go find GetInternals' signature.

Reply:
@twitchyliquid64, this wasn't done.